src/api/entities/SecurityToken/types.ts

```diff
@@ -48,9 +49,9 @@ export interface TransferBreakdown {
   result: boolean;
 }
 
-export interface ExternalAgent {
-  identity: Identity;
-  group: PermissionGroup;
+export interface AgentWithGroup {
+  agent: Agent;
+  group: PermissionGroupIdentifier;
```

Comment:
This should be a `KnownPermissionGroup | CustomPermissionGroup` like we discussed this morning